package.json

```diff
@@ -4,75 +4,78 @@
   "version": "1.0.2",
   "license": "apache-2.0",
   "scripts": {
-    "start": "webpack serve --host 0.0.0.0 --hot",
+    "start": "webpack-dev-server --host 0.0.0.0 --hot",
```

Comment:
What is the difference to `webpack serve`? Why is it worth the switch (and the extra dependency)?

Reply:
`serve` has actually been moved out of webpack in the latest versions, and the old webpack-serve package is deprecated. It's now succeeded by `webpack-dev-server` [here](https://github.com/webpack/webpack-cli/blob/1a8099c6e762249d05d3ec41478f23ad08deee4c/packages/serve/index.ts)

Reply:
Sg! Then let’s keep this :)